lib/ditcher.js

```diff
@@ -45,13 +46,83 @@ var Ditcher = function (cfg, lgr, versionNumber, callback) {
 
 util.inherits(Ditcher, EventEmitter);
 
+/**
+ * Creates an object containing the expression key and value queried.
+ *
+ * @param {String} expression - This is the operator used in query.
+ * @param {ObjectId|Number|Boolean} value - This is the value being quried.
+ *
+ * @returns {Object} expressionQuery - contains the expressions key and query value.
+ */
+function expressionQuery(expression, value) {
+  var expressionQuery = {};
+  expressionQuery[expression] = value;
+  return expressionQuery[expression];
```

Comment:
This looks like it only returns `value`. Shouldn't that be `return expressionQuery;`?